collectors/debugfs.plugin/debugfs_zswap.c

```diff
@@ -0,0 +1,117 @@
+    {.filename = "/sys/kernel/debug/zswap/stored_pages", .chart_id = "stored_pages",
+     .dimension = "page", .units = "pages", .title = "Compressed pages stored in zswap.",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_STORED_PAGE},
+    {.filename = "/sys/kernel/debug/zswap/pool_total_size", .chart_id = "pool_total_size",
+     .dimension = "pool", .units = "bytes", .title = "Total bytes used by the compressed storage",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_POOL_TOT_SIZE},
+    {.filename = "/sys/kernel/debug/zswap/duplicate_entry", .chart_id = "duplicate_entry",
+     .dimension = "duplicate", .units = "page", .title = "Duplicate store was found.",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_DUPP_ENTRY},
+    {.filename = "/sys/kernel/debug/zswap/written_back_pages", .chart_id = "written_back_pages",
+     .dimension = "pages", .units = "pages", .title = "Pages written back when pool limit was reached.",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_STORED_PAGE},
+    {.filename = "/sys/kernel/debug/zswap/reject_compress_poor", .chart_id = "reject_compress_poor",
+     .dimension = "pages", .units = "pages", .title = "Compressed page was too big for the allocator to store.",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_REJECT_COM_POOR},
+    {.filename = "/sys/kernel/debug/zswap/reject_kmemcache_fail", .chart_id = "reject_kmemcache_fail",
+     .dimension = "pages", .units = "pages", .title = "Number of entry metadata that could not be allocated",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_KMEM_FAIL},
+    {.filename = "/sys/kernel/debug/zswap/reject_alloc_fail", .chart_id = "reject_alloc_fail",
+     .dimension = "allocator", .units = "calls", .title = "Allocator could not get memory",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_RALLOC_FAIL},
+    {.filename = "/sys/kernel/debug/zswap/reject_reclaim_fail", .chart_id = "reject_reclaim_fail",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .dimension = "pages", .units = "pages", .title = "Memory cannot be reclaimed (pool limit was reached).",
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_RRECLAIM_FAIL},
+    {.filename = "/sys/kernel/debug/zswap/pool_limit_hit", .chart_id = "pool_limit_hit",
+     .enabled = CONFIG_BOOLEAN_YES, .chart_created = CONFIG_BOOLEAN_NO,
+     .dimension = "limit", .units = "boolean", .title = "Was the pool limit reached?",
+     .prio = NETDATA_CHART_PRIO_SYSTEM_ZSWAP_POOL_LIM_HIT},
```

Comment:
Are you sure this is a boolean and not the number of times the zswap pool limit has been reached?

Reply:
I arrived in this conclusion reading another project source code, bu after to read kernel source code I modified values, thanks!

Reply:
This has nothing to do with other projects and kernel. You guys agreed on formatting, see [clang-format](https://github.com/netdata/netdata/blob/master/.clang-format). I just asked to apply this format to this array.